dockstore-common/src/main/java/io/dockstore/common/DescriptorLanguage.java

```diff
@@ -32,13 +32,15 @@
     WDL("WDL", "Workflow Description Language", FileType.DOCKSTORE_WDL, FileType.WDL_TEST_JSON),
     // DOCKSTORE-2428 - demo how to add new workflow language
     //SWL("SWL", "Silly Workflow Language", FileType.DOCKSTORE_SWL, FileType.SWL_TEST_JSON)
-    NEXTFLOW("NFL", "Nextflow", FileType.NEXTFLOW_CONFIG, FileType.NEXTFLOW_TEST_PARAMS);
+    NEXTFLOW("NFL", "Nextflow", FileType.NEXTFLOW_CONFIG, FileType.NEXTFLOW_TEST_PARAMS),
+    SERVICE("service", "generic placeholder for services", FileType.DOCKSTORE_SERVICE_YML, FileType.DOCKSTORE_SERVICE_TEST_JSON);
```

Comment:
This seems a bit strange to me as a service isn't a type of language, but will itself have types. Is this just temporary for the transition to entry->(tool,workflow,service)?

Reply:
Depends a bit on where we go with the services proposal. 

But currently, services only have a yml like https://wiki.oicr.on.ca/display/DOC/Services+Proposal#ServicesProposal-Overview which from the webservice perspective is basically a descriptor.